.changeset/smooth-hairs-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-radar': minor
```

Comment:
think this is probably a patch release @rugvip?

Reply:
Yep, it is.

Reply:
Fully agree with you. Had patch level first, not sure why I changed it 😆 Updated!